pages/star-search/index.tsx

```diff
@@ -237,19 +255,32 @@ export default function StarSearchPage({ userId, ogImageUrl }: StarSearchPagePro
     inputRef.current.value = prompt;
     const { form } = inputRef.current;
 
-    if (typeof form.requestSubmit === "function") {
-      form.requestSubmit();
-    } else {
-      form.dispatchEvent(new Event("submit", { cancelable: true }));
+    if (bearerToken) {
+      if (typeof form.requestSubmit === "function") {
+        form.requestSubmit();
+      } else {
+        form.dispatchEvent(new Event("submit", { cancelable: true }));
+      }
     }
   }
 
+  function openWindow(url: string) {
```

Comment:
arrrgh, looks like that suggestion from StackOverflow doesn't work.

Reply:
Oh interesting, I tried it in Safari and Chrome and it worked for me

Reply:
I'm on Safari Version 16.6 (18615.3.12.11.2)

Reply:
I'm on Safari 17.5. Maybe something changed between those versions. 😅

Reply:
It works fine on Safari Desktop (I'm on 17.5), it's the iOS version that doesn't work.

Reply:
Ahh, ok 👀